assets/css/styles.css

```diff
@@ -4,29 +4,167 @@
   margin: 0;
   padding: 0;
   font-family: 'Poppins', sans-serif;
+  box-sizing: border-box;
 }
 
-.topHeader {
-  display: flex;
-  align-items: center;
-  justify-content: space-between;
-  padding: 3% 6.3% 0 6.2%;
-  box-sizing: border-box;
+html {
+  scroll-behavior: smooth;
+}
+
+header {
+  background-color: white;
 }
 
 .logo {
+  width: 50%;
+  height: 100%;
   font-style: normal;
   font-weight: 700;
-  font-size: 1.125rem;
-  line-height: 1.25rem;
-  color: #6070ff;
+  font-size: 18px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  background-color: #fff;
+  padding: 2px;
+}
+
+a {
+  color: #344563;
+  text-decoration: none;
 }
 
 .logo a {
+  margin-left: 24px;
+  background-color: #fff;
+  color: #5464f8;
   text-decoration: none;
-  color: #6070ff;
 }
 
+li {
+  list-style: none;
+}
+
+.nav-bar {
+  min-height: 70px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 24px;
+}
+
+.bar {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background-color: #5464f8;
+}
+
+.nav-bar .bar {
+  background-color: #5464f8;
+}
+
+.nav-menu {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  gap: 60px;
+  margin-right: 50px;
+}
+
+.nav-branding {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  color: #5464f8;
+  margin-left: 50px;
+}
+
+.nav-link {
+  transition: 0.7s ease;
+}
+
+.nav-link:hover {
+  color: #5464f8;
+}
+
+.hamburger {
+  display: none;
+  cursor: pointer;
+}
+
+@media (max-width: 768px) {
+  .hamburger {
+    display: block;
+  }
+
+  .bar {
+    background-color: white;
+  }
+
+  .hamburger.active .bar:nth-child(2) {
+    opacity: 0;
+    background-color: white;
+  }
+
+  .hamburger.active .bar:nth-child(1) {
+    transform: translateY(8px) rotate(45deg);
+    background-color: white;
+  }
+
+  .hamburger.active .bar:nth-child(3) {
+    transform: translateY(-8px) rotate(-45deg);
+    background-color: white;
+  }
+
+  .nav-bar.active {
+    background-color: #6070ff;
+  }
+
+  .nav-menu {
+    position: fixed;
+    left: -100%;
+    top: 0;
+    gap: 0;
+    flex-direction: column;
+    width: 100%;
+    height: 100%;
+    text-align: left;
+    transition: 0.3s;
+    background-color: #6070ff;
+    color: white;
+    align-items: flex-start;
+    justify-content: flex-start;
+    opacity: 90%;
```

Comment:
- [ OPTIONAL ] Great job for adding the opacity here but I think it would be great if you make it more blurred so you can make the opacity here  ` opacity: 107%;` to make it exactly like the figma so please fix it ✅🔥 I am assiging it as optional because you add it but it need more opacity✅💪🏻💯

- your
![image](https://user-images.githubusercontent.com/58553711/232951202-7e1458b2-518e-468b-81d0-396888462456.png)

- Figma 
![image](https://user-images.githubusercontent.com/58553711/232951240-5f5698a0-e1e5-4893-8974-b7469902de2b.png)